tests/ttnn/unit_tests/operations/test_moreh_matmul.py

```diff
@@ -95,19 +93,38 @@ def get_bias_tensors(bias_shape, require_bias_grad, device, use_int=True):
         if use_int
         else torch.rand(bias_shape, dtype=cpu_dtype) * 10 - 5
     )
-    tt_bias = ttnn.Tensor(bias, npu_dtype).pad_to_tile(float("nan")).to(npu_layout).to(device)
+    tt_bias = create_tt_tensor(bias, npu_dtype, npu_layout, device)
     tt_bias_grad = None
     if require_bias_grad:
         bias_grad = torch.full(bias_shape, float("nan"), dtype=cpu_dtype)
-        tt_bias_grad = ttnn.Tensor(bias_grad, npu_dtype).pad_to_tile(float("nan")).to(npu_layout).to(device)
+        tt_bias_grad = create_tt_tensor(bias_grad, npu_dtype, npu_layout, device)
     return tt_bias, bias, tt_bias_grad
 
 
-def moreh_matmul(params, has_output, compute_kernel_config, device):
+def moreh_matmul(
+    params,
+    has_output,
+    compute_kernel_config,
+    device,
+    use_randint=True,
+    npu_dtype=ttnn.bfloat16,
+    cpu_dtype=torch.bfloat16,
+    npu_layout=ttnn.TILE_LAYOUT,
+):
     torch.manual_seed(3072)
     input_shape, other_shape, output_shape, transpose_input, transpose_other = params
     tt_input, tt_other, tt_output, _, _, _, torch_input, torch_other, _ = get_tensors(
-        input_shape, other_shape, output_shape, False, False, False, device
+        input_shape,
+        other_shape,
+        output_shape,
+        False,
+        False,
+        False,
```

Comment:
To avoid confusion
```suggestion
        require_input_grad=False,
        require_other_grad=False,
        is_1d=False,
```

Reply:
This couldn't be done as they are positional args